<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="Test_Cases.Attempt_worksheet" tests="9" time="0.000" ignored="1" failures="0" timestamp="15 Dec 2017 09:02:27 GMT" skipped="8" hostname="net04-PC" errors="1">
  <testcase name="click_on_bookandworksheet_hidder" time="0.000" classname="Test_Cases.Attempt_worksheet">
    <skipped/>
  </testcase> <!-- click_on_bookandworksheet_hidder -->
  <testcase name="click_on_assignmenttype" time="0.000" classname="Test_Cases.Attempt_worksheet">
    <skipped/>
  </testcase> <!-- click_on_assignmenttype -->
  <testcase name="solve_worksheet" time="0.000" classname="Test_Cases.Attempt_worksheet">
    <skipped/>
  </testcase> <!-- solve_worksheet -->
  <testcase name="Openbrowser" time="0.000" classname="Test_Cases.Attempt_worksheet">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at configuration.TestNg_Config.Openbrowser(TestNg_Config.java:74)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:326)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Openbrowser -->
  <testcase name="click_on_worksheet" time="0.000" classname="Test_Cases.Attempt_worksheet">
    <skipped/>
  </testcase> <!-- click_on_worksheet -->
  <testcase name="click_on_chapter" time="0.000" classname="Test_Cases.Attempt_worksheet">
    <skipped/>
  </testcase> <!-- click_on_chapter -->
  <testcase name="start_worksheet" time="0.000" classname="Test_Cases.Attempt_worksheet">
    <skipped/>
  </testcase> <!-- start_worksheet -->
  <testcase name="Login" time="0.000" classname="Test_Cases.Attempt_worksheet">
    <skipped/>
  </testcase> <!-- Login -->
  <testcase name="select_class" time="0.000" classname="Test_Cases.Attempt_worksheet">
    <skipped/>
  </testcase> <!-- select_class -->
  <testcase name="navigateback" classname="Test_Cases.Attempt_worksheet">
    <skipped/>
  </testcase> <!-- navigateback -->
</testsuite> <!-- Test_Cases.Attempt_worksheet -->
